src/main/res/values/strings.xml

```diff
@@ -120,47 +120,47 @@
   <string name="saving_details">Saving details</string>
   <string name="screenshots">Screenshots</string>
   <string name="search">Search</string>
-  <string name="select_the_mirror">Select the mirror</string>
-  <string name="show_more">Show more</string>
-  <string name="show_older_releases">Show older releases</string>
+  <string name="select_the_mirror">Select a mirror</string>
+  <string name="show_more">More</string>
+  <string name="show_older_releases">Show older versions</string>
   <string name="skip">Skip</string>
   <string name="socks_proxy">SOCKS proxy</string>
-  <string name="sort_order">Sort order</string>
+  <string name="sort_order">Sorting order</string>
   <string name="source_code">Source code</string>
   <string name="suggested">Suggested</string>
   <string name="sync_repositories">Sync repositories</string>
-  <string name="sync_repositories_automatically">Sync repositories automatically</string>
-  <string name="syncing">Syncing</string>
-  <string name="syncing_format">Syncing %s</string>
+  <string name="sync_repositories_automatically">Automatically</string>
```

Comment:
"syncing" and "updating" are different processes. "Sync repositories automatically" should be used here.

Reply:
Isn't it already an option for "Sync repos"? Am I confused?

Reply:
No, "Sync repositories" is used as drop down menu button at the main screen.